### PR TITLE
New version: JetBrains.Gateway version 2023.3.4

### PR DESCRIPTION
**Pull request by SpecterShell**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in Run [#3391](https://github.com/SpecterShell/Dumplings/actions/runs/7920680624).
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/139263)

### CODE REVIEW

File: manifests/j/JetBrains/Gateway/2023.3.4/JetBrains.Gateway.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  Log: /LOG="<LOGPATH>"
+UpgradeBehavior: uninstallPrevious
+Protocols:
+- jetbrains-gateway
+ReleaseDate: 2024-02-15
+AppsAndFeaturesEntries:
+- DisplayName: JetBrains Gateway 2023.3.4
+  DisplayVersion: 233.14475.30
+  ProductCode: JetBrains Gateway 2023.3.4
+Installers:
```
Comment (stephengillie) [inline manifests/j/JetBrains/Gateway/2023.3.4/JetBrains.Gateway.installer.yaml:17]:
```suggestion
Scope: machine
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This was removed - should it be re-added?

**Comment by SpecterShell** [comment]
> This was removed - should it be re-added?

Nope. This can be installed in user scope by rejecting the UAC prompt at the beginning of the installation, and there is no command line switch to control this behavior. (There is a message file switch though)

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
> > This was removed - should it be re-added?
> 
> Nope. This can be installed in user scope by rejecting the UAC prompt at the beginning of the installation, and there is no command line switch to control this behavior. (There is a message file switch though)

@stephengillie

**Review by stephengillie — APPROVED** [review]

**Comment by TheStingPilot** [comment]
> > > This was removed - should it be re-added?
> > 
> > 
> > Nope. This can be installed in user scope by rejecting the UAC prompt at the beginning of the installation, and there is no command line switch to control this behavior. (There is a message file switch though)
> 
> @stephengillie

I think that is a missed opportunity. We use JetBrains and install it per machine. As we use  `--scope machine` in the install line, we cannot use WinGet for JetBrains. And installing per user is not an option as per machine is the default.

**Pull request merged**